tests/e2e/scenarios/lib/common.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+echo "CLOUD_PROVIDER=${CLOUD_PROVIDER}"
+echo "CLUSTER_NAME=${CLUSTER_NAME}"
+
+if [[ -n "${KOPS_BASE_URL-}" ]]; then
+    echo "KOPS_BASE_URL was unexpectedly set. Please unset."
+    exit 1
+fi
+
+export KOPS_FEATURE_FLAGS="SpecOverrideFlag,${KOPS_FEATURE_FLAGS:-}"
+export GO111MODULE=on
+
+if [[ -z "${AWS_SSH_PRIVATE_KEY_FILE-}" ]]; then
+    export AWS_SSH_PRIVATE_KEY_FILE="~/.ssh/id_rsa"
+fi
+if [[ -z "${AWS_SSH_PUBLIC_KEY_FILE-}" ]]; then
+    export AWS_SSH_PUBLIC_KEY_FILE="~/.ssh/id_rsa.pub"
+fi
+
+KUBETEST2="kubetest2 kops -v=2 --cloud-provider=${CLOUD_PROVIDER} --cluster-name=${CLUSTER_NAME:-}"
+KUBETEST2="${KUBETEST2} --admin-access=${ADMIN_ACCESS:-}"
+
+# Always tear-down the cluster when we're done
+function kops-finish {
+  ${KUBETEST2} --kops-binary-path="${KOPS}" --down || echo "kubetest2 down failed"
```

Comment:
I had to test this myself but can confirm that `KOPS` not being defined yet is acceptable here.